common/src/main/java/com/frostwire/bittorrent/BTDownload.java

```diff
@@ -130,7 +130,7 @@ public boolean isPaused() {
     }
 
     public boolean isSeeding() {
-        return th.isValid() && th.status().isSeeding();
+        return th.isValid() && th.status().isSeeding() && !isPaused();
```

Comment:
and this one might not even be necessary. `isSeeding()` should already return false if it's paused.

Will revert this changes and deal with this extra logic at the UI layer to not break anything.